spec/requests/admin/invitations_spec.rb

```diff
@@ -11,28 +11,28 @@
   describe "GET /admin/invitations" do
     it "renders to appropriate page" do
       user.update_column(:registered, false)
-      get "/admin/invitations"
```

Comment:
These paths stayed the same with the admin navigation restructure but I just changed them to use path helpers anyways